README.md

```diff
@@ -15,9 +15,62 @@ Carlo (Geant4 and Fluka) geometries.
 [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.10449301.svg)](https://doi.org/10.5281/zenodo.10449301)
 [![Read the Docs](https://img.shields.io/readthedocs/pyg4ometry?logo=readthedocs)](https://pyg4ometry.readthedocs.io)
 
+## Quick start
+
+_pyg4ometry_ is a very capable package to do many tasks related
+to Geant4/Fluka/MCNP geometry:
+
+- Python scripting to create and assemble geometries
+- Loading, editing and writing GDML
+- Load and tessellate CAD geometry and export to GDML
+- Load ROOT geometry and convert to GDML
+- Powerful VTK viewer of geometries
+- Converting from GDML to FLUKA and MCNP
+- Exporting mesh geometries from GDML to VTP, OBJ, VRML etc.
+- Python bindings to CGAL allowing complex mesh manipulation (e.g. hole filling, remeshing)
+
+All with few lines of Python code!
+
+```py
+import pyg4ometry as pg4
+from g4edgetestdata import G4EdgeTestData
+
+g4data = G4EdgeTestData()
+# define a geometry registry
+reg = pg4.geant4.Registry()
+
+# build the world volume
+# NOTE: syntax in the geant4 subpackage matches the Geant4 API!
+world_s = pg4.geant4.solid.Box("WorldAir", 10, 10, 10, reg, lunit="cm")
+world_l = pg4.geant4.LogicalVolume(world_s, "G4_AIR", "WorldAir", reg)
+reg.setWorld(world_l)
+
+# import an STL file
+reader = pg4.stl.Reader(g4data["stl/utah_teapot.stl"], reg)
```

Comment:
```suggestion
reader = pg4.stl.Reader(g4data["stl/utah_teapot.stl"], registry=reg)
```
the second (unnamed) argument is the solid name, thats why it crashed.

Reply:
Ok. Fixed.